float32 edgeConf
float32 confThreshold
float32 lostConf
float32 distance
